stable/gcloud-endpoints/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+1. Get the Endpoints URL by running:
```

Comment:
Can you add some more info on where people can go to find out more about Cloud Endpoints. Is there a sample app that you can point them to? Or maybe a tutorial on getting an app setup that they can leverage this chart for?